docs/index.rst

```diff
@@ -0,0 +1,20 @@
+.. pip-tools documentation master file, created by
+   sphinx-quickstart on Tue Jun 22 00:43:50 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to pip-tools's documentation!
+=====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   README
```

Comment:
This shouldn't be listed directly. Instead, include README from the project root dir.
```suggestion
```